### ISSUE TITLE
Missing accessibility label

### ISSUE THREAD

**Issue by TStArK1233** [signalapp/Signal-Android#12951]
![Screenshot_2023-05-17-19-46-35-966_org thoughtcrime securesms](https://github.com/signalapp/Signal-Android/assets/133884373/280f620d-add5-4233-93de-2cada022c520)

There is no lable to the button marked on red.  Thank You for fixing it😊😊.
